contracts/JBMultiTerminal.sol

```diff
@@ -691,8 +687,8 @@ contract JBMultiTerminal is JBOperatable, Ownable, ERC2771Context, IJBMultiTermi
                 _split.beneficiary != address(0) ? _split.beneficiary : payable(_msgSender());
```

Comment:
_msgSender() is the Terminal contract itself, no?

external call -> internal call -> external self call

Intended?

Reply:
oo interesting. is this caught in the test?